### PR TITLE
Add basics to student class

### PR DESCRIPTION
**Pull request by qiaoen17**

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -95,6 +97,16 @@ public interface Model {
 
     /////////////////////////
 
+    /** Returns an unmodifiable view of the filtered question list */
```
Comment (yuxuanleong) [inline src/main/java/seedu/address/model/Model.java:100]: This too 😄 "question list" to "student list"

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -107,6 +111,64 @@ public void removePerson(Person key) {
...
+     * The student must not already exist in the address book.
+     */
+    public void addStudent(Student s) {
+        students.add(s);
+    }
+
+    /**
+     * Replaces the given student {@code target} in the list with {@code editedStudent}.
+     * {@code target} must exist in the address book.
+     * The question identity of {@code editedStudent} must not be the same as another existing student in the address
```
Comment (yuxuanleong) [inline src/main/java/seedu/address/model/AddressBook.java:135]: Let's change 'question' identity change to 'student' identity

**Review by yuxuanleong — COMMENTED** [review]
All is good! Just some trivial changes needed for Javadocs

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (qiaoen17) [reply src/main/java/seedu/address/model/Model.java:100]: 👍

File: src/main/java/seedu/address/model/AddressBook.java (hunk 2 above)
Comment (qiaoen17) [reply src/main/java/seedu/address/model/AddressBook.java:135]: 👍

**Pull request merged**